### ISSUE TITLE
[VISUAL BUG] - Header Bar Overflow

### ISSUE THREAD

**Issue by sgoudham** [tommilligan/mdbook-admonish#20]
Hi 👋, just wanted to say that I really like this library and thank you for making it!

## Issue

However, I have noticed one thing though with the Header Bar in the given code blocks, they seem to _overflow_ and not line up. 

### Example

```rust
	```admonish <admonition type>
	[insert text]
	```
```

Results in:

![image](https://user-images.githubusercontent.com/58985301/165404957-e7711860-01a0-4e1e-be38-48473faa6203.png)
![image](https://user-images.githubusercontent.com/58985301/165405144-ba4c9686-be93-4ca4-84cc-a28dd3b5aded.png)
![image](https://user-images.githubusercontent.com/58985301/165404988-8151e34a-aeef-45da-9c02-6e99837f4e00.png)

It's like one or two pixels off so not entirely noticeable but definitely visible. I was wondering if this was an issue with the library or something potentially on the client-side? I've not changed any CSS within the generated CSS file.

## Diagnostic Info
Operating System: Windows 11
Rust: v1.58.1
mdbook: v0.4.18
mdbook-admonish: 1.5.0

**Comment by schungx** [comment]
It seems to have this problem on FireFox.

![image](https://user-images.githubusercontent.com/661318/165416745-c5a173ed-c286-4b54-8d69-c7618ed45662.png)

However, if I zoom in FireFox to, say, 150%, it disappears.

![image](https://user-images.githubusercontent.com/661318/165417161-223797be-f16c-4163-9f15-ae7bb228caf9.png)

However, Chrome doesn't have this problem.

![image](https://user-images.githubusercontent.com/661318/165416762-2f2c321f-64ba-4699-a7ae-7697d5f8d179.png)

However, MkDocs on FireFox seems to display correctly.

![image](https://user-images.githubusercontent.com/661318/165416898-487a057d-268c-47ce-9ad3-e18077ca4d6b.png)

Conclusion: It must be a conflict with `mdbook` styles.

**Comment by schungx** [comment]
I resolve the problem by commenting out these two lines in `mdbook-admonish.css`:

```css
:is(.admonition-title, summary) {
  position: relative;
  margin-block: 0;
  margin-inline: -1.6rem -1.2rem;
  padding-block: 0.8rem;
  padding-inline: 4rem 1.2rem;
  font-weight: 700;
  background-color: rgba(68, 138, 255, 0.1);
  border: 0 solid #448aff;
  /*
  border-inline-start-width: 0.4rem;
  border-start-start-radius: 0.2rem;
  */
  display: flex;
}
```

It seems to be a FireFox problem with sub-pixel placements.

**Comment by sgoudham** [comment]
> I resolve the problem by commenting out these two lines in `mdbook-admonish.css`:
> 
> ```css
> :is(.admonition-title, summary) {
>   position: relative;
>   margin-block: 0;
>   margin-inline: -1.6rem -1.2rem;
>   padding-block: 0.8rem;
>   padding-inline: 4rem 1.2rem;
>   font-weight: 700;
>   background-color: rgba(68, 138, 255, 0.1);
>   border: 0 solid #448aff;
>   /*
>   border-inline-start-width: 0.4rem;
>   border-start-start-radius: 0.2rem;
>   */
>   display: flex;
> }
> ```
> 
> It seems to be a FireFox problem with sub-pixel placements.

Ahhh interesting! Would commenting out these 2 lines affect how it's viewed on any other browsers?

**Comment by schungx** [comment]
> Ahhh interesting! Would commenting out these 2 lines affect how it's viewed on any other browsers?

Not that I can see... but I only tested Chrome and FireFox on desktop...

In fact, those two lines are inherited anyway, but setting it again for some reason causes FireFox to trip up...

**Comment by tommilligan** [comment]
Thanks both for the report and diagnosis - has made it super easy for me to implement a fix (see #21).

I think this also results in nicer code health all round, so thanks again :+1:
